examples/generate_report.js

```diff
@@ -0,0 +1,110 @@
+'use strict';
+
+let currencyCloud = require('../lib/currency-cloud');
+const opts = {
+    retries: 3,
+    factor: 2,
+    minTimeout: Math.random() * 750,
+    maxTimeout: Math.random() * 30000 + 30000,
+    randomize: true,
+    log: true
+};
+
+let generate_report = {
+    conversion_report_params: {
+        description: "Testing JS API",
+        buyCurrency: "EUR",
+        sellCurrency: "GBP",
+        uniqueRequestId: "q1w2e3r4"
```

Comment:
unique_request_id should be in UUIDv4 format. For example: e1821376-6cb0-4204-bec7-552733f3ea5a